main.nf

```diff
@@ -294,11 +387,33 @@ process insertsize {
     """
 }
 
+/*
+ * STEP 2.2 - Collapse
+ */
+process collapse {
+    label 'process_medium'
+    tag "$reads"
+    
+    input:
+    file reads from trimmed_reads_collapse
+
+    output:
+    file 'collapsed/*.fastq' into collapsed_fasta
+
+    script:
+    prefix = reads.toString() - '_trimmed.fq.gz'
+    """
+    seqcluster collapse -f $reads -m 1 --min_size 15 -o collapsed
```

Comment:
Is this the standard everyone would use in collapsing or should we make this configurable (the `--min_size 15`) parameter I mean?

Reply:
I can make it an option, but I this is pretty standard, the idea is not to lose any possible miRNA but not to add noise if the sample is bad.